aunt-leahs-app/src/pages/admin-locations.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
The file name should probably be adminLocations.jsx to follow convention

Reply:
Fixed, thanks!